crates/bevy_app/src/app.rs

```diff
@@ -448,7 +448,7 @@ impl App {
     {
         if !self.world.contains_resource::<Events<T>>() {
             self.init_resource::<Events<T>>().add_systems(
-                First,
+                FixedUpdate,
```

Comment:
Events are still very useful outside of traditional game applications; we should fallback / otherwise ensure that this works in apps where `FixedUpdate` doesn't exist.

Reply:
What would be a good way to detect/accommodate this?

Reply:
Maybe it'll work if `bevy_time` adds the resource.